configs/bundle_config.yaml

```diff
@@ -348,3 +348,13 @@ databundles:
     output: [cutouts/cutout-2013-era5.nc]
     disable_by_opt:
       build_cutout: [all]
+
+  bundle_cutouts_westernasia:
+    countries: [WAS]
+    category: cutouts
+    destination: "cutouts"
+    urls:
+      gdrive: https://drive.google.com/file/d/110UA0WQnmRV-20cXxYNR8WHLnLti6Qip/view?usp=drive_link
+    output: [cutouts/western-asia-2013-era5.nc]
```

Comment:
Could you please rename file to "cutout-2013-era5-tutorial.nc" to make it consistent with other cutouts names?